### PR TITLE
conditional broadcastUri

### PR DESCRIPTION
**Pull request by DaanHoogland**
### Description

This PR...
Fixes: #5596

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [ ] Minor
- [x] Trivial

### CODE REVIEW

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Review by weizhouapache — APPROVED** [review]
lgtm

File: ui/src/config/section/network.js [hunk 1]
```diff
@@ -29,7 +29,13 @@ export default {
...
       permission: ['listNetworks'],
       resourceType: 'Network',
-      columns: ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'broadcasturi', 'domain', 'account', 'zonename'],
+      columns: () => {
+        var fields = ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'domain', 'account', 'zonename']
+        if (['Admin'].includes(store.getters.userInfo.roletype)) {
+          fields.push('broadcasturi')
+        }
+        return fields
+      },
```
[the remark this answers is not in the document]
Comment (rhtyd) [reply ui/src/config/section/network.js:38]: Should we keep the same order, by pushing other fields after the if block?

[the remark this answers is not in the document]
Comment (DaanHoogland) [reply ui/src/config/section/network.js:38]: not sure, is that worth the trouble and the added code complexity?

**Comment by DaanHoogland** [comment]
@blueorangutan ui

**Review by sureshanaparti — APPROVED** [review]
code LGTM

File: ui/src/config/section/network.js [hunk 2]
```diff
@@ -29,8 +29,20 @@ export default {
       icon: 'apartment',
       permission: ['listNetworks'],
       resourceType: 'Network',
-      columns: ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'broadcasturi', 'domain', 'account', 'zonename'],
-      details: ['name', 'id', 'description', 'type', 'traffictype', 'vpcid', 'vlan', 'broadcasturi', 'cidr', 'ip6cidr', 'netmask', 'gateway', 'aclname', 'ispersistent', 'restartrequired', 'reservediprange', 'redundantrouter', 'networkdomain', 'zonename', 'account', 'domain'],
+      columns: () => {
+        var fields = ['name', 'state', 'type', 'vpcname', 'cidr', 'ip6cidr', 'broadcasturi', 'domain', 'account', 'zonename']
+        if (!['Admin'].includes(store.getters.userInfo.roletype)) {
```
Comment (sureshanaparti) [inline ui/src/config/section/network.js:34]: may be, this can be moved to a util method _isAdmin()_

```suggestion
        if (!isAdmin()) {
```

Comment (DaanHoogland) [reply ui/src/config/section/network.js:34]: makes sense but
```
$ grep -r isAdmin\ \(\)\ \{ ui/src/*
ui/src/views/compute/KubernetesServiceTab.vue:    isAdmin () {
ui/src/views/network/CreateNetwork.vue:    isAdmin () {
ui/src/views/network/UpdateNetwork.vue:    isAdmin () {
ui/src/views/network/CreateIsolatedNetworkForm.vue:    isAdmin () {
ui/src/views/network/CreateL2NetworkForm.vue:    isAdmin () {
ui/src/views/network/CreateSharedNetworkForm.vue:    isAdmin () {
ui/src/views/offering/AddComputeOffering.vue:    isAdmin () {
ui/src/views/offering/AddDiskOffering.vue:    isAdmin () {
ui/src/views/offering/AddNetworkOffering.vue:    isAdmin () {
ui/src/views/offering/AddVpcOffering.vue:    isAdmin () {
ui/src/views/offering/UpdateOfferingAccess.vue:    isAdmin () {
```
I think it should also be unified and made available in a generic way, than.

**Comment by sureshanaparti** [comment]
@blueorangutan ui

**Review by utchoang — APPROVED** [review]
LGTM!

**Comment by sureshanaparti** [comment]
Verified UI, Broadcast URI column not displayed for user.

**Comment by sureshanaparti** [comment]
Smoke tests not required for UI changes. Merging this based on the LGTMs, manual test and travis job.

**Pull request merged**